src/components/common/StudentAbout/_StudentAbout.scss

```diff
@@ -1,9 +1,132 @@
 @import '../../../styles/mixins';
 
-.div.component-StudentAbout {
+div.component-StudentAbout {
+  width: 250px;
   .student-about-wrap {
     @include flex(column, center, space-between);
-    width: 100px;
-    padding: 20px;
+    padding: 10px;
+    background: #292A2B;
```

Comment:
jeśli nie ma tego koloru w zmiennych, to dodaj do zmiennych i wstaw tu zmienną :)